.ai/skills/ai-add-prompt.md

```
@@ -0,0 +1,53 @@
+
+   ```markdown
+   ---
+   description: '<description>'
+   ---
+
+   #file:../../.ai/prompts/<name>.md
+   ```
+
+4. **If Claude active** — Create `.claude/skills/<name>.md`:
+
+   ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)